server/users/sql/index.ts

```diff
@@ -40,6 +40,30 @@ const createNewUser = `
 INSERT INTO users(firebase_id, invited_by, created_on)
 VALUES ($/firebase_id/, $/invited_by/, $/created_on/)`;
 
+const getUserRolesByRealm = `
+SELECT roles.*
```

Comment:
Indentation here :)

We don't have set rules but I generally try to indent the  `ON` and `AND` in the SQL, and the baseline of the query itself is also indented (as you can see above).